### ISSUE TITLE
关于调用cn_clip进行特征提取报错格式错误的问题

### ISSUE THREAD

**Issue by Seing-yu** [OFA-Sys/Chinese-CLIP#371]
Text的特征提取代码如下：
import cn_clip.clip as clip
from cn_clip.clip import load_from_name, available_models
class TextCLIPModel(nn.Module):

    def __init__(self, config, device):
        super().__init__()
        self.device = device
        self.model, self.preprocess = self._load_model(config)

    def _load_model(self, config):
        model, preprocess = load_from_name(config.clip_model_name, download_root=config.download_root)
        model.to(self.device)  # 将模型移动到指定设备
        model.eval()
        return model, preprocess

    def forward(self, texts):
        tokens = clip.tokenize(texts).to(self.device)
        with torch.no_grad():
            text_features = self.model.encode_text(tokens)
            text_features /= text_features.norm(dim=-1, keepdim=True)  # 归一化特征
        return text_features

![image](https://github.com/user-attachments/assets/9055d47e-a606-4a34-9ca7-e4dd0c218a9a)
输入的文本txt格式：
![image](https://github.com/user-attachments/assets/d8b5ad32-0d53-4604-a5f9-c84c0734a712)
输入的data.json格式如下：
![image](https://github.com/user-attachments/assets/ceaa555b-67a6-4bc0-a6a0-e1e98dd06ef2)
输入的img是data文件夹：
![image](https://github.com/user-attachments/assets/d1682c0e-7671-400b-a818-171c1c966c61)
我想问一下，这样输入数据是调用有问题，还是原始数据格式有问题
